iot-e2e-tests/android/buildAndRunTests.cmd

```diff
@@ -0,0 +1,7 @@
+call gradlew :app:assembleDebug :app:assembleDebugAndroidTest
+python AndroidDeviceSelect.py
+set /p device=<devcie.txt
+ECHO We're working with "%device%"
```

Comment:
devcie - > what is it ? is it saved somewhre? if so who creates it ? also please name it appropriately